src/Scorecard.js

```diff
@@ -0,0 +1,3 @@
+var Scorecard = function() {
```

Comment:
Right direction - focus on keeping everything together for now, and we can worry about extra classes/prototypes later